index.Rmd

```diff
@@ -262,13 +263,15 @@ How many roads are there in the resulting GeoDataFrame (hint: _len()_ or _.info(
 Load some building from another WFS service (BAG) and plot them too. How many buildings do you get? (hint: _len()_) Do you miss buildings? How can we extract missing buildings in our extent? 
 
 ```{Python, eval=FALSE}
+from matplotlib import pyplot as plt
 WfsUrl = 'https://geodata.nationaalgeoregister.nl/bag/wfs/v1_1'
 wfs = WebFeatureService(url=WfsUrl, version='1.1.0')
 layer = list(wfs.contents)[1]
 xmin, xmax, ymin, ymax = x-500, x+500, y-500, y+500
 response = wfs.getfeature(typename=layer, bbox=(xmin, ymin, xmax, ymax), outputFormat='json')
 data = json.loads(response.read())
 BuildingsGDF = gpd.GeoDataFrame.from_features(data['features'])
+BuildingsGDF.crs = 28992 # Set crs to RD New
```

Comment:
I assume the BAG WFS is in RD New, but not 100% sure. I got an error in the `NewBuildingsGDF.to_crs(epsg=3857)` in line 318) saying it could not convert the CRS from an undefined one.